elements/explicit.xml

```diff
@@ -25,7 +25,10 @@ type="application/xml" schematypens="http://purl.oclc.org/dsdl/schematron"?>
       <body xml:id="explicit">
          <div>
             <head>explicit</head>
-            <p/>
+            <p>Explicits are meant to give the user an impression of the texts included and help to navigate in a manuscript. They should 
```

Comment:
Some remarks. The wording "to give an impression" does not match the style of the guidelines. Quoting incipits and explicits serves the purpose of the description of the manuscript's content, and, specifically,  it is supposed 1) to help in identifying the texts or their parts; 2) to help in identifying the extention and completness of the texts or their parts in the manuscript.  I would say "Quotation should **preferebly** not include "common formulas or eulogies". I would not completely exclude the possibility for keeping formulaic expressions, as in some cases quoting initial formulas may be of interest or of importance (in older manuscripts they are different from the more recent). Also, sometimes the name of the commissioner can be mentioned in the formulaic part, and it may be better to encode a small piece of it, to keep te name.  I would also not say anything about later additions, since encoding added passages (if it somehow extends the original text?) as such is possible, the word "addition" can be interpreted in various ways.  
Another thing - if the text starts / ends _in abrupto_, we still quote the first words and call it "incipit" and "explicit", right? Should we mention these cases? (then quotation will be a little bit mechanical).
What I always do as cataloguer, I encode rubrications, but I don't know if we should ask for this. 
It is good to distinguish the point of view of the cataloguer (what I indicated before the point) and the pure user. But please check that the verb "to navigate" can be used in the way you do, I think "one navigates smth". Another thing - we speak about "explicit" (and better to keep it), does it interfere with "desinit"?

Reply:
>Some remarks. The wording "to give an impression" does not match the style of the guidelines. Quoting incipits and explicits serves the purpose of the description of the manuscript's content, and, specifically, it is supposed 1) to help in identifying the texts or their parts; 2) to help in identifying the extention and completness of the texts or their parts in the manuscript. I would say "Quotation should preferebly not include "common formulas or eulogies". I would not completely exclude the possibility for keeping formulaic expressions, as in some cases quoting initial formulas may be of interest or of importance (in older manuscripts they are different from the more recent). Also, sometimes the name of the commissioner can be mentioned in the formulaic part, and it may be better to encode a small piece of it, to keep te name. I would also not say anything about later additions, since encoding added passages (if it somehow extends the original text?) as such is possible, the word "addition" can be interpreted in various ways.

I tried a rework, that will hopefully meet all your quests. I also had some problems with the idea to avoid any kind of reliogious formulas.

Reply:
>What I always do as cataloguer, I encode rubrications, but I don't know if we should ask for this.

I agree, but I would not recommend to comment on that when describing incipit and explicit in a general way.

Reply:
>It is good to distinguish the point of view of the cataloguer (what I indicated before the point) and the pure user. But please check that the verb "to navigate" can be used in the way you do, I think "one navigates smth". Another thing - we speak about "explicit" (and better to keep it), does it interfere with "desinit"?

Personally, I used these terms interchangeably and I think their is no consensus about them, since both mean "end of a text" it may mean the end of the writing in a specific witness or the end of a work or the end of a planned writing process. I used the term desinit in my dissertation to describe the ever extended end of the text. For BM, I needed to change my terminology to "explicit". I am not aware, that the word desinit is used in BM/TEI in any context.